### ISSUE TITLE
Remove --pkg options since it was removed from pacman 5

### ISSUE THREAD

**Issue by discoltk** [oshazard/apacman#59]
Some AURs won't install, failing with this error:

makepkg: invalid option '--pkg'

It seems this option was removed from pacman.

**Comment by marcvangend** [comment]
I can confirm the error. For instance:

```
$ apacman -Syu
:: Synchronizing package databases...
 core is up to date
 extra is up to date
 community is up to date
 multilib is up to date
:: Starting full system upgrade...
 there is nothing to do
:: Synchronizing aur database...
 aur                                        95  95 [######################################################################]100%
:: Starting full aur upgrade...

Aur Targets    (3): libdbusmenu-glib libdbusmenu-gtk2 libdbusmenu-gtk3

:: Proceed with installation? [Y/n] 

/tmp/pkgbuild-1000/libdbusmenu-glib
  -> Split package base: libdbusmenu
  -> PKGBUILD contains: pkgbase-glib pkgbase-gtk23
Edit libdbusmenu-glib PKGBUILD with $editor? [Y/n] n
makepkg: invalid option '--pkg'

==> ERROR: the build failed
  -> Status	failed (1): libdbusmenu-glib
/tmp/pkgbuild-1000/libdbusmenu-gtk2
  -> Split package base: libdbusmenu
  -> PKGBUILD contains: pkgbase-glib pkgbase-gtk23
Edit libdbusmenu-gtk2 PKGBUILD with $editor? [Y/n] n
makepkg: invalid option '--pkg'

==> ERROR: the build failed
  -> Status	failed (2): libdbusmenu-glib libdbusmenu-gtk2
/tmp/pkgbuild-1000/libdbusmenu-gtk3
  -> Split package base: libdbusmenu
  -> PKGBUILD contains: pkgbase-glib pkgbase-gtk23
Edit libdbusmenu-gtk3 PKGBUILD with $editor? [Y/n] n
makepkg: invalid option '--pkg'

==> ERROR: the build failed
  -> Status	failed (3): libdbusmenu-glib libdbusmenu-gtk2 libdbusmenu-gtk3

 local database is up to date
```

**Comment by oshazard** [comment]
Resolved in #61
